interface/main/holidays/import_holidays.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+//this part download the CSV file after the click on the href link
+if($_GET['download_file']==1)
+{
+    if ( ! file_exists($_GET['filename']))
+    {
+        echo 'file missing';
+    }
+    else
+    {
+        header('HTTP/1.1 200 OK');
+        header('Cache-Control: no-cache, must-revalidate');
+        header("Pragma: no-cache");
+        header("Expires: 0");
+        header("Content-type: text/csv");
+        header("Content-Disposition: attachment; filename='holiday.csv'");
+        readfile($_GET['filename']);
+        exit;
+    }
+    die();
+}
+// end download section
+
+?>
+<?php
+/**
+ *
+ * @package OpenEMR
+ * @author  sharon cohen <sharonco@matrix.co.il>
+ * @link    http://www.open-emr.org
+ */
+
+set_time_limit(0);
+
+// Disable magic quotes and fake register globals.
+$sanitize_all_escapes = true;
+$fake_register_globals = false;
+require_once('../../globals.php');
+require_once($GLOBALS['srcdir'] . '/acl.inc');
+require_once("$srcdir/htmlspecialchars.inc.php");
+require_once("$srcdir/sql.inc");
+require_once("Holidays_Controller.php");
+
+
+//require_once($GLOBALS['srcdir'] . '/htmlspecialchars.inc.php');
+//require_once($GLOBALS['srcdir'] . '/formdata.inc.php');
+//require_once($GLOBALS['fileroot'] . '/custom/code_types.inc.php');
+
+if (!acl_check('admin', 'super')) die(xlt('Not authorized'));
+
+//$form_replace = !empty($_POST['form_replace']);
+//$code_type = empty($_POST['form_code_type']) ? '' : $_POST['form_code_type'];
+
+// Handle uploads.
+$holidays_controller = new Holidays_Controller();
+$csv_file_data = $holidays_controller->get_file_csv_data();
+
+if (!empty($_POST['bn_upload'])) {
+    //Upload and save the csv
+    $saved = $holidays_controller->upload_csv($_FILES);
+}
+if (!empty($_POST['import_holidays'])) {
+    //Import from the csv file to the calendar external table
+    $saved = $holidays_controller->import_holidays_from_csv();
+}
+if (!empty($_POST['sync'])) {
+    //Upload and save the csv
+    $saved = $holidays_controller->create_holiday_event();
+}
+
+
+?>
+
+<html>
+<head>
+    <title><?php echo xlt('Holidays management '); ?></title>
+    <link rel="stylesheet" href='<?php echo $css_header ?>' type='text/css'>
+
+</head>
+
+<body class="body_top">
+<?php
+if ($saved){
+    echo "<p style='color:green'>" .
+        xlt('Successfully Completed ');
```

Comment:
Don't place trailing (or leading) spaces within translation constants. Place them outside the constant.